superset-frontend/src/components/Select/styles.tsx

```diff
@@ -272,6 +294,24 @@ export const DEFAULT_COMPONENTS: SelectComponentsConfig<any> = {
       />
     </DropdownIndicator>
   ),
+  Input: (props: InputProps) => (
+    <div style={{ position: 'relative' }}>
+      <Input {...props} />
+      {!!(props.selectProps.isMulti && props.selectProps.value.length) && (
+        <span
+          style={{
+            color: '#879399',
+            position: 'absolute',
+            top: '2px',
+            left: '4px',
+            width: '100vw',
```

Comment:
Looking for suggestions as to the best place to put these styles.

Reply:
in an emotion `styled.span`?

Reply:
ok, I put the styles in a const.. seems to be a common pattern? 
I didn't see a way to pull a color in from a pallet, though, so it's hard-coded unless there's any advice there.